### PR TITLE
adding lite to the release automation

### PR DESCRIPTION
**Pull request by chayim**
Also added in to the nightly automation area of CircleCI, in case it gets used in the future

### CODE REVIEW

File: tests/qa/RS_VERSIONS-lite [hunk 1]
```diff
@@ -0,0 +1,2 @@
+6.0.12-58
+100.0.0-2400
```
Comment (alonre24) [inline tests/qa/RS_VERSIONS-lite:2]: This should be 100.0.0-2440 isn't it? @tomerhekredis ?

File: tests/qa/RS_VERSIONS [hunk 2]
```diff
@@ -1,2 +1 @@
-6.0.12-58
 100.0.0-2400
```
Comment (alonre24) [inline tests/qa/RS_VERSIONS:1]: This should be 100.0.0-2440 isn't it? @tomerhekredis ?

File: tests/qa/run [hunk 3]
```diff
@@ -89,10 +90,12 @@ run_test() {
 }
 
 rc=0
+# By default we use RS_VERSIONS, but with variants they may need to specify their own redis verisons, hence this file
+VERSIONFILE=${HERE}/RS_VERSIONS${VARIANT}
```
Comment (alonre24) [inline tests/qa/run:94]: We send `VARIANT=lite` from `circleci.yaml` but the file name is `RS_VERSIONS-lite` (with a dash) - is that a problem?

**Review by alonre24 — COMMENTED** [review]
Few comments

File: tests/qa/run (hunk 3 above)
Comment (chayim) [reply tests/qa/run:94]: CircleCI passes in VARIANT=-lite, which we use to build the file. This was validated with @tomerhekredis

File: tests/qa/RS_VERSIONS-lite (hunk 1 above)
Comment (chayim) [reply tests/qa/RS_VERSIONS-lite:2]: On non-lite @tomerhekredis said we wanted 100.0.0-2400 on on RS_VERSIONS-<variant> the 6.0x thing.  It's worth noting we can now basically do anything here by adding variants and files. @tomerhekredis ?

File: tests/qa/RS_VERSIONS (hunk 2 above)
Comment (chayim) [reply tests/qa/RS_VERSIONS:1]: As above.

File: tests/qa/RS_VERSIONS-lite (hunk 1 above)
Comment (tomerhekredis) [reply tests/qa/RS_VERSIONS-lite:2]: yes, it should be 2440 in all RS_VERSIONS files

**Review by alonre24 — APPROVED** [review]

**Pull request merged**